src/main/kotlin/com/cognifide/gradle/aem/pkg/ExtractPackageTask.kt

```diff
@@ -0,0 +1,73 @@
+package com.cognifide.gradle.aem.pkg
+
+import com.cognifide.gradle.aem.api.AemDefaultTask
+import com.cognifide.gradle.aem.base.vlt.VltCleaner
+import com.cognifide.gradle.aem.internal.file.FileException
+import org.gradle.api.tasks.Input
+import org.gradle.api.tasks.Internal
+import org.gradle.api.tasks.TaskAction
+import java.io.File
+
+open class ExtractPackageTask : AemDefaultTask() {
+
+    companion object {
+        const val NAME = "aemDownloadExtract"
+        const val FORCE_NEW_FLAG = "extract.force.new"
```

Comment:
no `aem.` prefix

Reply:
Added aem.download prefix to configuration flags